### DISCUSSION TITLE
Comments on getting started section and salat_polish_tseries.pro

### DISCUSSION THREAD

**Discussion by guadabsb15** [SolarAlma/SALAT#2] [General]
In `installation.md`,  we could add a note about adding **SALAT/IDL** to path, in addition to what it is written with something like: on Linux `export IDL_PATH=+/pathtosalat/SALAT/IDL:$IDL_PATH` 

Also, on `installation.md` it would be useful to change command `.r SALAT` with `.r salat`, since the name of the file is lowercase in this package and it won't work otherwise   

If there are additional packages needed, it could be a good idea to say something about it here. For example, it looks like `salat_polish_tseries.pro` needs the routine `red_shc.pro` . I found it on the following repo [colinsauze/crispred](https://github.com/colinsauze/crispred). I am not sure if that repo is needed for SALAT, but if it is, we should add a note about it. 

Another comment, it seems like there is an extra comma at the end of line 51 in `salat_polish_tseries.pro` 

https://github.com/SolarAlma/SALAT/blob/f2bb95928b62146439ba6ceb2a5309ae21fc8ce1/IDL/salat_polish_tseries.pro#L51-L52

I will be adding a few more comments as appropriate on this thread.

**Comment by guadabsb15** [comment]
Also the parameter `cubic=cubic` gives a syntax error. It seems like it is not being defined 
https://github.com/SolarAlma/SALAT/blob/f2bb95928b62146439ba6ceb2a5309ae21fc8ce1/IDL/salat_polish_tseries.pro#L52

https://github.com/SolarAlma/SALAT/blob/f2bb95928b62146439ba6ceb2a5309ae21fc8ce1/IDL/salat_polish_tseries.pro#L148

https://github.com/SolarAlma/SALAT/blob/f2bb95928b62146439ba6ceb2a5309ae21fc8ce1/IDL/salat_polish_tseries.pro#L240

https://github.com/SolarAlma/SALAT/blob/f2bb95928b62146439ba6ceb2a5309ae21fc8ce1/IDL/salat_polish_tseries.pro#L252

**Reply by shahin-jafarzadeh** [reply]
It is likely the lack of libraries/dependencies. I will look into it and will add all missing dependencies. Thanks for pointing this out. We should rather make the package independent of external libraries.

**Comment by guadabsb15** [comment]
After removing the `cubic =cubic` (for debugging purposes) from the above lines, the error I get now is about `readfits` not being defined. 
https://github.com/SolarAlma/SALAT/blob/f2bb95928b62146439ba6ceb2a5309ae21fc8ce1/IDL/salat_polish_tseries.pro#L186

Is that routine in another package/library I should be adding? 

Any comments on this will be appreciated :)

**Reply by shahin-jafarzadeh** [reply]
This again be the lack of libraries, though `readfits` is one of the most used functions in IDL. Will add this, along with other missing dependencies, later today.

**Comment by shahin-jafarzadeh** [comment]
Thanks for your suggestion on adding information for IDL path, etc. The link provided has the instruction. THE `export IDL_PATH=+/pathtosalat/SALAT/IDL:$IDL_PATH` is indeed one solution, but only for bash environment on unix-based machines. 

The missing function will be added, thanks.

You are right about the extra comma and is being removed.
